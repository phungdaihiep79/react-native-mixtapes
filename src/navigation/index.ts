export { default } from './navigation'
export * from './type'
export * from './NavigationService'
