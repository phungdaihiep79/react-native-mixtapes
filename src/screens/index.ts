export { Home } from './Home'
export { Hell } from './Hell'
export { Heaven } from './Heaven'
export { AppEbikeFake } from './AppEbikeFake'
export { Hell2 } from './Hell2'
