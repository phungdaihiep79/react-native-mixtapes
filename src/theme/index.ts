export * from './nomalize'
export * from './color'
export * from './font'
export * from './style'
