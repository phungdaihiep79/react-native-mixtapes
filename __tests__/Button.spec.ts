import 'react-native';
import React from 'react';
import {create} from 'react-test-renderer';
import {Button} from '../src/components';

describe('Button Component', () => {
  test('render correctly', () => {
    // @ts-ignore
    const tree = create(Button).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
