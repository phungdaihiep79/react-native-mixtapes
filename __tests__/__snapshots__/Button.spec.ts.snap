// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component render correctly 1`] = `null`;
